apisix/plugins/opa.lua

```diff
@@ -125,6 +127,11 @@ function _M.access(conf, ctx)
         end
 
         return status_code, reason
+    else if conf.send_header_upstream and result.headers then
+           for key,value in pairs(result.headers) do
+                core.request.set_header(ctx,key,value)
+           end
+        end
```

Comment:
Is it necessary to proxy all of opa's request headers?

Reply:
What else do you propose? How to select which ones to proxy?

Reply:
what HTTP header should pass upstream? instead of forwarding all HTTP header